src/commands/playbook.js

```diff
@@ -0,0 +1,13 @@
+import loadCommand from '../util/loadCommand'
+import composeInvoke from '../util/composeInvoke'
+
+export const {parse, usage, commandDescriptor} = loadCommand('playbook')
+
+export const invoke = composeInvoke(parse, usage, (args, notify, options) => {
+  const {
+    formatMessage,
+  } = options
+
+  notify(formatMessage('📖  Opening Playbook'))
```

Comment:
It seems to me that, if the user passes a "search string", our message should be something like "Searching Playbook for '<search string>'" rather than "Opening Playbook".